cssom-view-1/Overview.bs

```diff
@@ -1335,16 +1335,21 @@ a <a>scrolling box</a> <var>scrolling box</var>, in order of innermost to outerm
 1. Let <var>scrolling box edge D</var> be the <a>ending edge</a> in the <a>inline base direction</a> of <var>scrolling box</var>, and let <var>element edge D</var> be <var>element bounding border box</var>'s edge on the same physical side as that of <var>scrolling box edge D</var>.
 1. Let <var>element width</var> be the distance between <var>element edge C</var> and <var>element edge D</var>.
 1. Let <var>scrolling box width</var> be the distance between <var>scrolling box edge C</var> and <var>scrolling box edge D</var>.
-1. If <var>scrollMode</var> is "<code>if-hidden</code>", and <var>element</var> is partially or entirely in view already, then return.
-
-        ISSUE: Define "partially or entirely in view".
-
 1. If <var>block</var> is undefined, set <var>block</var> to a UA-defined value.
 1. If <var>inline</var> is undefined, set <var>inline</var> to a UA-defined value.
 
         ISSUE: Define defaults for <var>block</var> and <var>inline</var> for {{FocusScrollOptions}}.
 
-1. Let <var>position</var> be the scroll position <var>scrolling box</var> would have by following these steps:
+1. Let <var>hidden</var> be false.
+1. If <var>element edge B</var> is outside (before) <var>scrolling box edge A</var>, or
```

Comment:
It seems that 'if-hidden' works only when an element is entirely outside the scrolling box, but I think the coverage of 'if-hidden' would be two cases like 'entirely out of view' and 'partially in view', while 'always' should support the two cases plus an 'entirely in view' as the three cases totally.

In other words, "if-hidden" means scrolling only if an element is hidden (partially or entirely out the view).

The cases like 'scrollIntoViewIfNeeded' method and 'notIfViewed' property works as above now.
https://lists.w3.org/Archives/Public/www-style/2012May/0808.html

In terms of real use cases, 'if-hidden' seems to work as above.

Reply:
I also agree with 'if-hidden' works when an element is entirely or partially out of the view.
Sorry for the confusion!

Reply:
OK, I must have misunderstood @anawhj earlier comment.